clouddatabasesv5/cloud_databases_v5_integration_test.go

```diff
@@ -227,14 +234,20 @@ var _ = Describe(`CloudDatabasesV5 Integration Tests`, func() {
 		})
 		It(`ChangeUserPassword(changeUserPasswordOptions *ChangeUserPasswordOptions)`, func() {
 
+			var userName string = "user"
+
+			if strings.Contains(deploymentID, "postgresql") || strings.Contains(deploymentID, "enterprisedb") {
+				userName = "repl"
+			}
```

Comment:
This seems specific to testing Logical Replication slot. maybe move updating the `repl` user password into that test and leave this as-is